src/components/MapView/AlertForm/index.tsx

```diff
@@ -0,0 +1,354 @@
+                  >
+                    <MenuItem
+                      style={{ color: 'black' }}
+                      key="placeholder"
+                      value="placeholder"
+                      disabled
+                    >
+                      Choose a statistic
+                    </MenuItem>
+                    <MenuItem
+                      style={{ color: 'black' }}
+                      key="minbelow"
+                      value="minbelow"
+                    >
+                      Minimum Below
+                    </MenuItem>
+                    <MenuItem
+                      style={{ color: 'black' }}
+                      key="maxabove"
+                      value="maxabove"
+                    >
+                      Maximum Above
+                    </MenuItem>
+                  </Select>
+                </FormControl>
+              </div>
+              <div className={classes.AlertFormOptions}>
+                <TextField
+                  id="threshold"
+                  className="thresholdClass"
+                  label="Threshold"
+                  type="number"
+                  variant="filled"
+                  defaultValue={0}
+                  onChange={e =>
+                    setSelectedThreshold(parseInt(e.target.value, 10))
+                  }
+                />
+              </div>
+              <div className={classes.AlertFormOptions}>
+                <Select
+                  id="regionsList"
+                  className="regionsListClass"
+                  label="Regions to monitor"
+                  type="text"
+                  variant="filled"
+                  value={regionsList}
+                  multiple
+                  style={{ maxWidth: '200px' }}
+                  onChange={e => {
+                    const selected: string[] = e.target.value as string[];
+                    const lastSelected = selected[selected.length - 1];
+
+                    if (lastSelected !== 'allRegions') {
+                      setRegionsList(
+                        selected.filter(el => el !== 'allRegions'),
+                      );
+                    } else {
+                      // If 'allRegions' was the last selected then it should be the only thing selected.
+                      setRegionsList(['allRegions']);
+                    }
+                  }}
+                >
+                  [
+                  <MenuItem
+                    style={{ color: 'black' }}
+                    key="allRegions"
+                    value="allRegions"
+                  >
+                    All
+                  </MenuItem>
+                  <ListSubheader>Administrative Regions</ListSubheader>
+                  ...
+                  {regionNames.map(region => {
+                    return (
+                      <MenuItem
+                        style={{ color: 'black' }}
```

Comment:
I noticed when I wrote `LayerDropdown` I was doing the same thing, and styling this as black. Maybe we can style this once and for all as black in the global MUI style, and remove all the inline styles both here and in `layerdropdown`?